README.md

```diff
@@ -299,8 +299,8 @@ The following arguments are used to configure the mode:
 
 If the flag --persistent_file_storage_location is not used then loading occurs without writing a temporary file.
 
-if the value for the flag --persistent_file_storage_location is specified in the format gs://project-id/bucket-id/some-object,
- the file will be written to the specified project bucket. It is also necessary to set the system variable [GOOGLE_APPLICATION_CREDENTIALS](https://cloud.google.com/healthcare/docs/how-tos/authentication#setting_the_environment_variables) = path_to_file_with_json_data.
+if the value for the flag --persistent_file_storage_location is specified in the format gs://bucket-id/some-object,
```

Comment:
I think it would be more accurate to say "gs://bucket-id/some-directory" rather than "gs://bucket-id/some-object", since this is really a folder where we're writing the files to. Some of our official docs already use this 'directory' terminology: https://cloud.google.com/healthcare/docs/reference/rest/v1beta1/projects.locations.datasets.fhirStores/import#gcssource